Assets/GameAssets/Player/PlayerScripts/PlayerHealth.cs

```diff
@@ -43,8 +48,40 @@ public override void IncreaseHealth(float heal)
         base.IncreaseHealth(heal);
 
         PlayerManager.Instance.InvokePlayerHealthChangeEvent(GetHealthPercent(), _currentHealth);
+        CheckHeartBeat();
     }
 
+    private void CheckHeartBeat()
```

Comment:
Suggestion: This can be called `PlayHeartBeatSfx` and the XML comment can specify it will only play if the health is within the specified threshold

In my _opinion_, `CheckHeartBeat` implies it should be returning a bool and is ONLY a check, when in reality it does more